ska_helpers/tests/test_git_helpers.py

```diff
@@ -21,11 +21,19 @@ def ska_ownership_ok():
         return False
 
 
+def git_version_old():
+    # Is the git version too old to have the repo_safe feature?
+    # This is the version of git not gitpython.
+    repo = git.Repo(CHANDRA_MODELS)
+    return repo.git.version_info < (2, 35, 2)
```

Comment:
What's the recommended way to compare versions these days?

Reply:
I think it's `packaging.version`

Reply:
You probably *could* do:
```
import packaging

git_version = '.'.join(str(val) for val in repo.git.version_info)
old = packaging.version(git_version) < packaging.version("2.35.2")
```

Reply:
I think you don't need the repo, this is enough:
```
git.Git.version_info
```